src/playgroundrl_envs/games/petting_zoo.py

```diff
@@ -93,13 +93,23 @@ def get_is_game_over(self):
     @staticmethod
     def get_game_name():
         return "petting_zoo"
+    
+    @staticmethod
+    def get_game_subname(self):
+        return self.game_name
 
     @staticmethod
     def get_num_players():
         return 2
 
     def get_outcome(self, player_id):
-        pass
-
+        if not self.is_game_over:   
+            return None
+        if self.is_game_over and self.reward == -1:      
```

Comment:
You don't need this first clause, since it'll return if self.is_game_over. I.e., you can just do 
```
if self.reward == -1:
```
Also, I don't think we ever set self.reward, so we'll need to change this. The desired functionality here is that it'll return an outcome specific to player_id. So for example, if we player_id's 0, and 1, player 0 won and player 1 lost, then get_outcome(0) should return 1, but get_outcome(1) should return 0.